lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -1,4 +1,6 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
+    expose :validate_import_vm
+    expose :import_vm
```

Comment:
Should we override the return or is there any useful object that can be returned so the user can track the progress?

```ruby
expose : import_vm,   :override_return => true
```
Example: [lib/miq_automation_engine/service_models/miq_ae_service_miq_request.rb#L13](https://github.com/ManageIQ/manageiq/blob/fd5e8acce71d53b28360eeb0167eed0f151e7df3/lib/miq_automation_engine/service_models/miq_ae_service_miq_request.rb#L13)